plugin-dev/Source/Sentry/Private/Tests/SentryEvent.spec.cpp

```diff
@@ -15,7 +17,8 @@ void SentryEventSpec::Define()
 {
 	BeforeEach([this]()
 	{
-		SentryEvent = NewObject<USentryEvent>();
+		TSharedPtr<ISentryEvent> NativeImpl = MakeShareable(new FGenericPlatformSentryEvent);
+		SentryEvent = USentryEvent::Create(NativeImpl);
```

Comment:
Do we need to capture the shared ptr?
```suggestion
		SentryEvent = USentryEvent::Create(MakeShareable(new FGenericPlatformSentryEvent));
```